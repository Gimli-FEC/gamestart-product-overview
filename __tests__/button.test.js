import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';

import Button from '../client/components/button.jsx';

afterEach(cleanup);

test('renders', () => {
  const { asFragment } = render(<Button />);
  expect(asFragment()).toMatchSnapshot();
});
