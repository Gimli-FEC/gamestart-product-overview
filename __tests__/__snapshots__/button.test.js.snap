// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <button
    class="button__AddToCart-sc-1vptnfg-0 dVUQAg"
  >
    Add to cart
  </button>
</DocumentFragment>
`;
